turbopack/crates/turbo-persistence/Cargo.toml

```diff
@@ -14,6 +14,7 @@ print_stats = ["stats"]
 anyhow = { workspace = true }
 pot = "3.0.0"
 byteorder = "1.5.0"
+chrono = "0.4.40"
```

Comment:
Apparently the hot new time library everyone is excited about is https://github.com/BurntSushi/jiff